src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/Intellisense.cs

```diff
@@ -32,11 +33,12 @@ public Intellisense(PowerFxConfig config, IEnumStore enumStore, IReadOnlyList<IS
             _suggestionHandlers = suggestionHandlers;
         }
 
+        [SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "n/a")]
```

Comment:
>n/a [](http://example.com/codeflow?start=99&length=3)

We can have something like `No need to crash the process in case of exception during intellisense; instead we return an empty result set along with exception for client use.` here.